src/plugins/intel_cpu/src/nodes/proposal.h

```diff
@@ -7,8 +7,8 @@
 #include <ie_common.h>
 #include <node.h>
 #include "proposal_imp.hpp"
```

Comment:
minor: Since Proposal cross compilation is no longer required I would propose to unite `proposal` and `proposal_imp` files

Reply:
done